### PR TITLE
system variable: rename tidb_ddl_distribute_reorg to tidb_enable_dist_task

### PR DESCRIPTION
**Pull request by Benjamin2037**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
修改系统参数 tidb_ddl_distribute_reorg 为 tidb_enable_dist_task。

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -978,18 +978,18 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
-### `tidb_ddl_distribute_reorg` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+### `tidb_enable_dist_task` <span class="version-mark">从 v7.1.0 版本开始引入</span>
 
-> **警告：**
->
-> - 该功能目前为实验特性。不推荐在生产环境中开启该功能。
-> - 当前启用此功能后，在 DDL reorg 阶段遇到某些异常只会做简单重试，还没有兼容 DDL 操作的重试方式，即目前无法依据  [`tidb_ddl_error_count_limit`](#tidb_ddl_error_count_limit) 的大小控制重试次数。
+> **注意：**
+> 
+> - 该参数是从 `tidb_ddl_distribute_reorg`参数改名而来。
```
Comment (ran-huang) [inline system-variables.md:985]:
```suggestion
> - 该参数是从 `tidb_ddl_distribute_reorg` 参数改名而来。
```

File: system-variables.md [hunk 2]
```diff
@@ -978,18 +978,18 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+> - 该功能目前为实验特性。
+> - 当前启用此功能后，DDL, Import 等后端任务将会启动分布式执行任务的方式来利用集群中的多个 TiDB 节点来共同完成任务。
```
Comment (ran-huang) [inline system-variables.md:987]:
```suggestion
> - 当前启用此功能后，DDL、Import 等后端任务将会启动分布式执行任务的方式来利用集群中的多个 TiDB 节点来共同完成任务。
```

File: system-variables.md [hunk 3]
```diff
@@ -978,18 +978,18 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 
 - 作用域：GLOBAL
 - 是否持久化到集群：是
 - 默认值：`OFF`
-- 这个变量用于控制是否开启分布式执行 DDL reorg 阶段，来提升此阶段的速度。目前此开关只对 `ADD INDEX` 语句有效。开启该变量对于数据量较大的表有一定的性能提升。分布式 DDL 会通过 DDL 动态资源管控，控制 DDL 的 CPU 使用量，来防止对线上业务产生影响。
-- 要验证已经完成的 `ADD INDEX` 操作是否使用了此功能，可以查看 `mysql.tidb_background_subtask_history` 表是否有对应任务。
+- 这个变量用于控制是否开启分布式执行。开启后 DDL，Import 等支持的任务将会由集群中多个 TiDB 节点共同完成。
```
Comment (ran-huang) [inline system-variables.md:992]:
```suggestion
- 这个变量用于控制是否开启分布式执行。开启分布式执行后，DDL、Import 等支持的任务将会由集群中多个 TiDB 节点共同完成。
```

Comment (ran-huang) [inline system-variables.md:992]: 这里比较模糊，能否明确说下具体哪些任务是支持的？

File: system-variables.md [hunk 4]
```diff
@@ -978,18 +978,19 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 >
 > 目前，PITR 恢复会额外处理日志备份时间段内通过索引加速功能创建的索引，以达到兼容效果。详细内容请参考[索引加速功能为什么与 PITR 功能不兼容](/faq/backup-and-restore-faq.md#索引加速功能为什么与-pitr-功能不兼容)。
 
-### `tidb_ddl_distribute_reorg` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+### `tidb_enable_dist_task` <span class="version-mark">从 v7.1.0 版本开始引入</span>
```
Comment (ran-huang) [inline system-variables.md:982]:
```suggestion

> **警告：**
>
> 该功能目前为实验特性，不建议在生产环境中使用。
```

File: system-variables.md [hunk 5]
```diff
@@ -978,18 +978,19 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+> - 该参数是从 `tidb_ddl_distribute_reorg` 参数改名而来。
+> - 该功能目前为实验特性。
+> - 当前启用此功能后，DDL、Import 等后端任务将会启动分布式执行任务的方式来利用集群中的多个 TiDB 节点来共同完成任务。
 
 - 作用域：GLOBAL
 - 是否持久化到集群：是
 - 默认值：`OFF`
-- 这个变量用于控制是否开启分布式执行 DDL reorg 阶段，来提升此阶段的速度。目前此开关只对 `ADD INDEX` 语句有效。开启该变量对于数据量较大的表有一定的性能提升。分布式 DDL 会通过 DDL 动态资源管控，控制 DDL 的 CPU 使用量，来防止对线上业务产生影响。
-- 要验证已经完成的 `ADD INDEX` 操作是否使用了此功能，可以查看 `mysql.tidb_background_subtask_history` 表是否有对应任务。
+- 这个变量用于控制是否开启分布式执行。开启分布式执行后，DDL、Import 等支持的任务将会由集群中多个 TiDB 节点共同完成。
```
Comment (ran-huang) [inline system-variables.md:993]:
```suggestion
- 这个变量用于控制是否开启分布式执行框架。开启分布式执行后，DDL、Import 等支持的后端任务将会由集群中多个 TiDB 节点共同完成。
- 在 TiDB v7.1.0 中，只支持分布式执行分区表的 `ADD INDEX`。
- 该参数由 `tidb_ddl_distribute_reorg` 参数改名而来。
```

File: system-variables.md [hunk 6]
```diff
@@ -978,18 +978,19 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
-> **警告：**
->
-> - 该功能目前为实验特性。不推荐在生产环境中开启该功能。
-> - 当前启用此功能后，在 DDL reorg 阶段遇到某些异常只会做简单重试，还没有兼容 DDL 操作的重试方式，即目前无法依据  [`tidb_ddl_error_count_limit`](#tidb_ddl_error_count_limit) 的大小控制重试次数。
+> **注意：**
+> V7.1 只提供分区表 `Add index` 的分布式执行支持。
+> 
+> - 该参数是从 `tidb_ddl_distribute_reorg` 参数改名而来。
+> - 该功能目前为实验特性。
+> - 当前启用此功能后，DDL、Import 等后端任务将会启动分布式执行任务的方式来利用集群中的多个 TiDB 节点来共同完成任务。
```
Comment (ran-huang) [inline system-variables.md:988]:
```suggestion
```

**Review by zimulala — COMMENTED** [review]
LGTM

**Review by zimulala — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: releases/release-6.6.0.md [hunk 7]
```diff
@@ -75,7 +75,7 @@ TiDB 版本：6.6.0-[DMR](/releases/versioning.md#开发里程碑版本)
 
 * 支持 DDL 分布式并行执行框架（实验特性）[#37125](https://github.com/pingcap/tidb/issues/37125) @[zimulala](https://github.com/zimulala)
 
-    在过去的版本中，整个 TiDB 集群中仅允许一个 TiDB 实例作为 DDL Owner 处理 Schema 变更任务。为了进一步提升 DDL 的并发性，TiDB v6.6.0 版本引入了 DDL 分布式并行执行框架，支持集群中所有的 TiDB 实例并发执行同一个任务的 `StateWriteReorganization` 阶段，加速 DDL 的执行。该功能由系统变量 [`tidb_ddl_distribute_reorg`](/system-variables.md#tidb_ddl_distribute_reorg-从-v660-版本开始引入) 控制是否开启，目前只支持 `Add Index` 操作。
+    在过去的版本中，整个 TiDB 集群中仅允许一个 TiDB 实例作为 DDL Owner 处理 Schema 变更任务。为了进一步提升 DDL 的并发性，TiDB v6.6.0 版本引入了 DDL 分布式并行执行框架，支持集群中所有的 TiDB 实例并发执行同一个任务的 `StateWriteReorganization` 阶段，加速 DDL 的执行。该功能由系统变量 `tidb_ddl_distribute_reorg` 控制是否开启，目前只支持 `Add Index` 操作。
```
Comment (qiancai) [inline releases/release-6.6.0.md:78]:
```suggestion
    在过去的版本中，整个 TiDB 集群中仅允许一个 TiDB 实例作为 DDL Owner 处理 Schema 变更任务。为了进一步提升 DDL 的并发性，TiDB v6.6.0 版本引入了 DDL 分布式并行执行框架，支持集群中所有的 TiDB 实例并发执行同一个任务的 `StateWriteReorganization` 阶段，加速 DDL 的执行。该功能由系统变量 [`tidb_ddl_distribute_reorg`](https://docs.pingcap.com/zh/tidb/v6.6/system-variables#tidb_ddl_distribute_reorg-从-v660-版本开始引入) 控制是否开启，目前只支持 `Add Index` 操作。
```

File: releases/release-6.6.0.md [hunk 8]
```diff
@@ -147,7 +147,7 @@ TiDB 版本：6.6.0-[DMR](/releases/versioning.md#开发里程碑版本)
 
 * 支持 DDL 动态资源管控（实验特性）[#38025](https://github.com/pingcap/tidb/issues/38025) @[hawkingrei](https://github.com/hawkingrei)
 
-    TiDB v6.6.0 版本引入了 DDL 动态资源管控，通过自动控制 DDL 的 CPU 使用量，尽量降低 DDL 变更任务对线上业务的影响。该功能仅在开启 [DDL 分布式并行执行框架](/system-variables.md#tidb_ddl_distribute_reorg-从-v660-版本开始引入)后生效。
+    TiDB v6.6.0 版本引入了 DDL 动态资源管控，通过自动控制 DDL 的 CPU 使用量，尽量降低 DDL 变更任务对线上业务的影响。该功能仅在开启 DDL 分布式并行执行框架后生效。
```
Comment (qiancai) [inline releases/release-6.6.0.md:150]:
```suggestion
    TiDB v6.6.0 版本引入了 DDL 动态资源管控，通过自动控制 DDL 的 CPU 使用量，尽量降低 DDL 变更任务对线上业务的影响。该功能仅在开启 [DDL 分布式并行执行框架](https://docs.pingcap.com/zh/tidb/v6.6/system-variables#tidb_ddl_distribute_reorg-从-v660-版本开始引入)后生效。
```

File: system-variables.md [hunk 9]
```diff
@@ -978,18 +978,18 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
-> - 该功能目前为实验特性。不推荐在生产环境中开启该功能。
-> - 当前启用此功能后，在 DDL reorg 阶段遇到某些异常只会做简单重试，还没有兼容 DDL 操作的重试方式，即目前无法依据  [`tidb_ddl_error_count_limit`](#tidb_ddl_error_count_limit) 的大小控制重试次数。
+> 该功能目前为实验特性，不建议在生产环境中使用。
 
 - 作用域：GLOBAL
 - 是否持久化到集群：是
 - 默认值：`OFF`
-- 这个变量用于控制是否开启分布式执行 DDL reorg 阶段，来提升此阶段的速度。目前此开关只对 `ADD INDEX` 语句有效。开启该变量对于数据量较大的表有一定的性能提升。分布式 DDL 会通过 DDL 动态资源管控，控制 DDL 的 CPU 使用量，来防止对线上业务产生影响。
-- 要验证已经完成的 `ADD INDEX` 操作是否使用了此功能，可以查看 `mysql.tidb_background_subtask_history` 表是否有对应任务。
+- 这个变量用于控制是否开启分布式执行框架。开启分布式执行后，DDL、Import 等支持的后端任务将会由集群中多个 TiDB 节点共同完成。
```
Comment (qiancai) [inline system-variables.md:990]:
```suggestion
- 这个变量用于控制是否开启分布式执行框架。开启分布式执行后，DDL 和 Import 等后端任务将会由集群中多个 TiDB 节点共同完成。
```

File: system-variables.md [hunk 10]
```diff
@@ -978,18 +978,18 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+- 在 TiDB v7.1.0 中，只支持分布式执行分区表的 `ADD INDEX`。
+- 该参数由 `tidb_ddl_distribute_reorg` 参数改名而来。
```
Comment (qiancai) [inline system-variables.md:992]:
```suggestion
- 该变量由 `tidb_ddl_distribute_reorg` 改名而来。
```

File: system-variables.md [hunk 11]
```diff
@@ -978,18 +978,18 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
-> - 当前启用此功能后，在 DDL reorg 阶段遇到某些异常只会做简单重试，还没有兼容 DDL 操作的重试方式，即目前无法依据  [`tidb_ddl_error_count_limit`](#tidb_ddl_error_count_limit) 的大小控制重试次数。
+> 该功能目前为实验特性，不建议在生产环境中使用。
 
 - 作用域：GLOBAL
 - 是否持久化到集群：是
 - 默认值：`OFF`
-- 这个变量用于控制是否开启分布式执行 DDL reorg 阶段，来提升此阶段的速度。目前此开关只对 `ADD INDEX` 语句有效。开启该变量对于数据量较大的表有一定的性能提升。分布式 DDL 会通过 DDL 动态资源管控，控制 DDL 的 CPU 使用量，来防止对线上业务产生影响。
-- 要验证已经完成的 `ADD INDEX` 操作是否使用了此功能，可以查看 `mysql.tidb_background_subtask_history` 表是否有对应任务。
+- 这个变量用于控制是否开启分布式执行框架。开启分布式执行后，DDL 和 Import 等后端任务将会由集群中多个 TiDB 节点共同完成。
+- 在 TiDB v7.1.0 中，只支持分布式执行分区表的 `ADD INDEX`。
```
Comment (qiancai) [inline system-variables.md:991]: 在 TiDB v7.1.0 中，除了 `ADD INDEX`，其他 DDL 和 Import 等后端任务可以由集群中多个 TiDB 节点共同完成吗？

Comment (Benjamin2037) [reply system-variables.md:991]: Yes，后续目标就是这样的。

**Review by qiancai — APPROVED** [review]

**Pull request merged**